libparsec/crates/platform_mountpoint/src/memfs.rs

```diff
@@ -146,16 +154,13 @@ impl MountpointInterface for MountpointManager {
             .ok_or(MountpointError::NotFound)?;
         let id = parent
             .children
-            .remove(
-                &EntryName::try_from(
-                    source
-                        .file_name()
-                        .expect("Contains a name")
-                        .to_str()
-                        .expect("Non utf8 name"),
-                )
-                .expect("Should be a valid EntryName"),
-            )
+            .remove(&EntryName::try_from(
+                source
+                    .file_name()
+                    .expect("Contains a name")
+                    .to_str()
+                    .expect("Non utf8 name"),
```

Comment:
```suggestion
                    .map_err(|_| MountpointError::InvalidName)?,
```

Otherwise the fs can be crashed just by passing a non-utf8 path...

Reply:
Done